pkgs/by-name/co/cosmic-session/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  fetchFromGitHub,
+  bash,
+  rustPlatform,
+  just,
+  dbus,
+  rust,
+  stdenv,
+  xdg-desktop-portal-cosmic,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-session";
+  version = "0-unstable-2023-01-14";
```

Comment:
`version = "0-unstable-2023-01-12"` since the latest commit is on January 12th instead of 14th, PR title and commit msg needs to reflect that as well.

Reply:
I would also assume `2024-01-12` instead of `2023` for the given commit.

Reply:
> I would also assume `2024-01-12` instead of `2023` for the given commit.

yep